components/bluetooth/lib.rs

```diff
@@ -777,10 +804,12 @@ impl BluetoothManager {
         return drop(sender.send(Err(BluetoothError::NotFound)));
     }
 
+    // https://webbluetoothcg.github.io/web-bluetooth/#dom-bluetoothremotegattservice-getincludedservices
```

Comment:
We really need to link to https://webbluetoothcg.github.io/web-bluetooth/#getgattchildren somewhere as well.